examples/Patterns/Saga/Runner.cs

```diff
@@ -34,8 +34,10 @@ public Runner(int numberOfIterations, double uptime, double refusalProbability,
 
         private PID CreateAccount(string name, Random random)
         {
-            var accountProps = Actor.FromProducer(() => new Account(name, _uptime, _refusalProbability, _busyProbability, random));
-            return Actor.SpawnNamed(accountProps, name);
+            var accountProps = Props.FromProducer(() => new Account(name, _uptime, _refusalProbability, _busyProbability, random));
+            //TODO: I've never used ProtoActor before, so don't know is this is an appropropriate fix to access a global static Context
+            // perhaps should be injected somehow during proto construction?
+            return Program.Context.SpawnNamed(accountProps, name);
```

Comment:
accessing global static Context feels wrong. If this can please be checked, would be much appreciated! txs